neqo-transport/src/connection.rs

```diff
@@ -3694,4 +3734,181 @@ mod tests {
         let c_tx_dgrams = send_bytes(&mut client, 0, now);
         assert_eq!(c_tx_dgrams.len(), 4);
     }
+
+    fn check_is_packet_drop(peer: &mut Connection, pkt: Option<Datagram>, dropped: u64, dups: u64) {
```

Comment:
It looks like this is used for both duplicates or drops, so call this `check_discarded()`.